lms/djangoapps/certificates/tests/test_webview_views.py

```diff
@@ -947,6 +968,55 @@ def test_html_view_certificate_available_date_for_instructor_paced_courses(self,
         )
         self.assertContains(response, date)
 
+    @override_settings(FEATURES=FEATURES_WITH_CERTS_ENABLED)
+    @ddt.data(
+        (True, False),
+        (False, False),
+        (True, True),
+        (False, True)
+    )
+    @ddt.unpack
+    def test_html_view_certificate_display_date(self, self_paced, date_override):
+        """
+        Test certificate web view should display the correct date on the
+        certificate in all cases:
+            * self-paced, no date override
+            * instructor-paced with certificate_available_date
+            * self-paced with date override
+            * instructor-paced with date override
+        """
```

Comment:
This is my new test. I want to make sure the correct date ends up displayed on the certificate, in all our various cases. This is loosely modeled after the test right above it, but more specific to what I want to test.